medium/profiles/models.py

```diff
@@ -12,28 +12,28 @@ class Colors(models.TextChoices):
         WHITE = "#FFFFFF", _("White")
 
     class Meta:
-        verbose_name = _("Profiles")
+        verbose_name = _("Profile")
         ordering = ["-created_at"]
 
     user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
+    about_text = models.TextField()
+    profile_pic = models.ImageField(upload_to=get_profile_pic_url)
+    header_pic = models.ImageField(upload_to=get_header_pic_url)
```

Comment:
stick to the naming scheme if there is any field name doesn't make sense
make sure to change the field name in schema first then reflect changes in the implementation.

Reply:
Both fields point to and Image field not a URL field so the name was misleading, we can change our version of the schema to solve the confusion